js/plugins/views.js

```diff
@@ -0,0 +1,73 @@
+var clinicalTimelineViewHandler = function(options) {
```

Comment:
I think you should get rid of this function and move the init/remove code to the corresponding places in the plugins.